rotkehlchen/db/dbhandler.py

```diff
@@ -2768,27 +2768,49 @@ def get_latest_asset_value_distribution(self) -> List[DBAssetBalance]:
             ignored_assets = self.get_ignored_assets(write_cursor)
 
         with self.conn.read_ctx() as cursor:
+            treat_eth2_as_eth = self.get_settings(cursor).treat_eth2_as_eth
             cursor.execute(
                 'SELECT time, currency, amount, usd_value, category FROM timed_balances WHERE '
                 'time=(SELECT MAX(time) from timed_balances) AND category = ? ORDER BY '
                 'CAST(usd_value AS REAL) DESC;',
                 (BalanceType.ASSET.serialize_for_db(),),  # pylint: disable=no-member
             )
             asset_balances = []
+            eth_balance: Dict[str, str] = {
+                'time': '0',
+                'amount': '0',
+                'usd_value': '0',
+            }
             for result in cursor:
                 asset = Asset(result[1])
                 if asset in ignored_assets:
                     continue
+                # show eth & eth2 as eth in value distribution by asset
+                if treat_eth2_as_eth is True and asset in (A_ETH, A_ETH2):
+                    eth_balance['time'] = str(result[0])
+                    eth_balance['amount'] = str(FVal(eth_balance['amount']) + FVal(result[2]))
+                    eth_balance['usd_value'] = str(FVal(eth_balance['usd_value']) + FVal(result[3]))  # noqa: E501
+                else:
+                    asset_balances.append(
+                        DBAssetBalance(
+                            time=result[0],
```

Comment:
isn't the linter complaining here about missing a casting to `Timestamp`?

Reply:
It has a type of `Any`